README.md

```diff
@@ -44,6 +44,16 @@ JdPay.qr_pri_key = 'YOUR_RSA_PRIVATE_KEY'
 #JdPay.debug_mode = true # Enable parameter check. Default is true.
 ```
 
+*重要*：
+
+根据文档中的Q&A
+> 支付请求和异步返回的加密规则是什么？
+  答：支付请求用pkcs8的RSA私钥进行签名，des加密；
+    同步返回使用京东支付统一对外rsa公钥验签；
+    异步返回用3DES解密，京东支付统一对外rsa公钥验签。
+
+因此请勿为`JdPay.pub_key`设置你自己产生的公钥，默认不填就行。
```

Comment:
```ruby
# JdPay.pub_key = 'JDPAY_RSA_PUBLIC_KEY'
```
It's enough.

Reply:
I beleive it's not clear enough? For the new comers, they could misunderstand this `JDPAY_RSA_PUBLIC_KEY ` key. Add comments could help them have a better understanding.